lib/reply.js

```diff
@@ -244,6 +248,47 @@ Reply.prototype.headers = function (headers) {
   return this
 }
 
+// https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Trailer#directives
+// https://httpwg.org/specs/rfc7230.html#chunked.trailer.part
+const INVALID_TRAILERS = new Set([
+  'transfer-encoding',
+  'content-length',
+  'host',
+  'cache-control',
+  'max-forwards',
+  'te',
+  'authorization',
+  'set-cookie',
+  'content-encoding',
+  'content-type',
+  'content-range',
+  'trailer'
+])
+
+Reply.prototype.trailer = function (key, fn) {
+  const _key = key.toLowerCase()
```

Comment:
Why declare a new variable? Reassign is enough :)
```suggestion
  key = key.toLowerCase()
```